src/main/java/ch/njol/skript/doc/JSONGenerator.java

```diff
@@ -372,6 +390,18 @@ private static JsonArray cleanPatterns(String... strings) {
 		return convertToJsonArray(strings);
 	}
 
+	private void cacheEvents() {
```

Comment:
you should probably move this to a static block so it doesnt get run multiple times which could add the same event info multiple times